instrumentation/github.com/go-kit/kit/otelkit/config.go

```diff
@@ -106,3 +109,10 @@ func WithAttributeGetter(fn func(ctx context.Context) []attribute.KeyValue) Opti
 		o.GetAttributes = fn
 	})
 }
+
+// WithSpanKind sets the span kind for spans created by this middleware.
```

Comment:
Please document the default value.

```suggestion

// WithSpanKind sets the span kind for spans created by this middleware.
// If this option is not provided, then trace.SpanKindServer is used.

Reply:
Done :+1: